src/components/data/DataTable.js

```diff
@@ -359,9 +359,9 @@ const DataTable = props => {
                       }, [
                         h(HeaderCell, [
                           !!columnNamespace && span({ style: { fontStyle: 'italic', color: colors.dark(0.75), paddingRight: '0.2rem' } },
-                            columnNamespace)
-                        ]),
-                        [columnName]
+                            columnNamespace),
+                          columnName
+                        ])
```

Comment:
`columnName` was moved outside of the HeaderCell in #2314 / [AS-503](https://broadworkbench.atlassian.net/browse/AS-503), but is not necessary for the goal of that change.